index.js

```diff
@@ -8,23 +8,24 @@ const core = require('@actions/core')
 
 module.exports = app => {
   app.on('push', async context => {
+    let exitEarly = false
     const config = await getConfig({
       app,
       context,
       getConfig: require('probot-config')
+    }).catch(error => {
+      log({ app, context, error, message: 'Invalid config file' })
+      exitEarly = true
     })
 
+    if (exitEarly) return
+
     // GitHub Actions merge payloads slightly differ, in that their ref points
     // to the PR branch instead of refs/heads/master
     const ref = process.env['GITHUB_REF'] || context.payload.ref
 
     const branch = ref.replace(/^refs\/heads\//, '')
 
-    if (!config.template) {

```

Comment:
added as a required object in the Joi schema so it should throw an error if deep merge does not contain `template`